.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!--
+Thank you for your pull request. Please provide a description above and
+review the checklist below.
+
+Contributors guide: ./CONTRIBUTING.md
+-->
+
```

Comment:
A `\## Description` header might also be useful.

Reply:
Could you please clarify what you mean here? I'm afraid I don't understand

Reply:
I meant that we have an 'area' in the template for contributor to provide a description of what the PR is supposed to achieve.